common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1884,6 +1884,15 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     WRITE_SET_REAPER_INTERVAL("hive.writeset.reaper.interval", "60s",
       new TimeValidator(TimeUnit.MILLISECONDS), "Frequency of WriteSet reaper runs"),
 
+    // For Druid storage handler
+    HIVE_DRUID_DEFAULT_ADDRESS("hive.druid.address.default", "", "Default address of the Druid broker"),
```

Comment:
Can you improve description of this config? Is it the address used in create table statement if address is not specified in table properties? Also, this is not used for select queries where address is retrieved from metastore which stores fully qualified broker address?